package.json

```diff
@@ -21,36 +21,35 @@
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "^2.7.14",
+    "vue": "3.3.4",
```

Comment:
Are you in favour of locking down this version, or should we use `^` to auto-update to minor and patch versions when they become available?

Same question for vuelidate, below. (I know it was that way before, but what are your thoughts about it now?)

Reply:
I did a bit of reading and thinking about this. It should be okay to use `^` to auto-update

Reply:
OK. We'll need to keep an eye on it when they release new minor/patch versions and we rebuild the package lock file (which is when packages auto-update).
👍